docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -691,6 +691,8 @@ Default stratifications to all observers should include scenario and input draw.
       * Delivery facility
       * Azithromycin coverage
       * Misoprostol coverage 
+      * Hemoglobin screening coverage
+      * Ferritin screening coverage
```

Comment:
I think we don't need to add these as additional strata because they don't have any direct effect on maternal disorders